raiden/transfer/node.py

```diff
@@ -745,6 +746,14 @@ def handle_receive_unlock(
     return subdispatch_to_paymenttask(chain_state, state_change, secrethash)
 
 
+def handle_update_transport_synctoken(
+        chain_state: ChainState,
```

Comment:
I don't think `ChainState` is the right abstraction to store this -- `transport` and `blockchain` are two pretty orthogonal concepts, and I am not convinced we do ourselves a service with coupling them.

I will spend some thought on what other existing state container fits. Maybe we should now take on the docstring TODO:
`TODO: Split the node specific attributes to a "NodeState" class`